pages/cart/cart.css

```diff
@@ -0,0 +1,66 @@
+@import url(../../basic.css);
+main {
+    margin-top: 2rem;
+}
+
+.cart-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    max-width: 65rem;
+    margin: 0 auto;
+}
+
+.card {
+    padding: 0.5rem 0;
+}
+
+.card-btn {
+    width: 100%;
+    padding: 0.4rem;
+    background-color: var(--lightblue);
+    color: var(--white);
+    border: none;
+    margin-top: 1rem;
+    cursor: pointer;
+}
+
+.price-detail {
+    padding: 0.5rem;
+    border-top: 1px solid gray;
+    border-bottom: 1px solid gray;
+    margin: 0.5rem 0;
+}
+
+.price-detail-list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
use this in the class directly from the basic.css you have defined instead of using it here